### PR TITLE
Document automaticUserConsentSettings in XTAP

### PR DESCRIPTION
**Pull request by jkdouglas**

### CODE REVIEW

**Comment by jkdouglas** [comment]
API review pr link: https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/11482

**Comment by FaithOmbongi** [comment]
Hi @jkdouglas - I see from the related schema (Krunal's schema PR # 6711944) that this feature is still in private preview. I'm putting this PR in the content review backlog and will review it when your team is ready to release the feature to public preview.

image

**Comment by jkdouglas** [comment]
@FaithOmbongi - here is the PR to unhide the schema. Can you help review this now?

https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/7422721

File: api-reference/beta/api/crosstenantaccesspolicyconfigurationpartner-update.md [hunk 1]
```diff
@@ -54,17 +54,18 @@ PATCH /policies/crossTenantAccessPolicy/partners/{id}
...
+| automaticUserConsentSettings | [inboundOutboundPolicyConfiguration](../resources/inboundoutboundpolicyconfiguration.md) | Determines the partner-specific configuration for automatic user consent settings. |
 
 ## Response
 
 If successful, this method returns a `204 No Content` response code.
 
 ## Examples
 
-### Request
+### Example 1: Configure inbound trust settings to accept MFA, compliant, and Hybrid Azure AD Joined devices from a partner organization
```
Comment (FaithOmbongi) [inline api-reference/beta/api/crosstenantaccesspolicyconfigurationpartner-update.md:66]: I suggest shortening the title and instead adding a description to give context for the example that follows.
```suggestion
### Example 1: Configure inbound trust settings

The following example configures the partner-specific policy by setting the inbound trust settings to accept MFA, compliant, and Hybrid Azure AD Joined devices from the partner tenant.

```

File: api-reference/beta/api/crosstenantaccesspolicyconfigurationpartner-update.md [hunk 2]
```diff
@@ -85,34 +86,42 @@ Content-Type: application/json
...
-[!INCLUDE [sample-code](../includes/snippets/java/update-crosstenantaccesspolicyconfigurationpartner-java-snippets.md)]
-[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+``` http
+HTTP/1.1 204 No Content
+```
 
-# [Go](#tab/go)
-[!INCLUDE [sample-code](../includes/snippets/go/update-crosstenantaccesspolicyconfigurationpartner-go-snippets.md)]
-[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+### Example 2: Configure automaticUserConsent settings to consent for B2B collaboration on behalf of your users and accept admin consent for another partner tenant's users
```
Comment (FaithOmbongi) [inline api-reference/beta/api/crosstenantaccesspolicyconfigurationpartner-update.md:102]: Same suggestion as previous
```suggestion
### Example 2: Configure automaticUserConsent settings 

The following example configures the partner-specific policy by consenting for B2B collaboration on behalf of your users and accepting admin consent for the partner's users.

```

File: api-reference/beta/resources/crosstenantaccesspolicyconfigurationdefault.md [hunk 3]
```diff
@@ -32,6 +32,7 @@ The default configuration defined for inbound and outbound settings of Azure AD
 | b2bDirectConnectInbound  |[crossTenantAccessPolicyB2BSetting](../resources/crosstenantaccesspolicyb2bsetting.md) | Defines your default configuration for users from other organizations accessing your resources via Azure AD B2B direct connect. |
 | b2bDirectConnectOutbound | [crossTenantAccessPolicyB2BSetting](../resources/crosstenantaccesspolicyb2bsetting.md) |Defines your default configuration for users in your organization going outbound to access resources in another organization via Azure AD B2B direct connect. |
 | inboundTrust | [crossTenantAccessPolicyInboundTrust](../resources/crosstenantaccesspolicyinboundtrust.md) | Determines the default configuration for trusting other Conditional Access claims from external Azure AD organizations. |
+| automaticUserConsentSettings | [inboundOutboundPolicyConfiguration](../resources/inboundoutboundpolicyconfiguration.md) | Determines the default configuration for automatic user consent settings. `inboundAllowed` and `outboundAllowed` will always be false and cannot be updated in the default configuration. **Read only**. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/crosstenantaccesspolicyconfigurationdefault.md:35]: Move to top of table

File: api-reference/beta/api/crosstenantaccesspolicyconfigurationpartner-update.md [hunk 4]
```diff
@@ -54,17 +54,18 @@ PATCH /policies/crossTenantAccessPolicy/partners/{id}
 | b2bDirectConnectInbound | [crossTenantAccessPolicyB2BSetting](../resources/crosstenantaccesspolicyb2bsetting.md) | Defines your partner-specific configuration for users from other organizations accessing your resources via Azure AD B2B direct connect. |
 | b2bDirectConnectOutbound | [crossTenantAccessPolicyB2BSetting](../resources/crosstenantaccesspolicyb2bsetting.md) | Defines your partner-specific configuration for users in your organization going outbound to access resources in another organization via Azure AD B2B direct connect. |
 | inboundTrust | [crossTenantAccessPolicyInboundTrust](../resources/crosstenantaccesspolicyinboundtrust.md) | Determines the partner-specific configuration for trusting other Conditional Access claims from external Azure Active Directory (Azure AD) organizations. |
+| automaticUserConsentSettings | [inboundOutboundPolicyConfiguration](../resources/inboundoutboundpolicyconfiguration.md) | Determines the partner-specific configuration for automatic user consent settings. |
```
Comment (FaithOmbongi) [inline api-reference/beta/api/crosstenantaccesspolicyconfigurationpartner-update.md:57]: Move to top of table

File: api-reference/beta/api/crosstenantaccesspolicy-post-partners.md [hunk 5]
```diff
@@ -56,6 +56,7 @@ The following table shows the properties that are required when you create the [
 | b2bDirectConnectInbound | [crossTenantAccessPolicyB2BSetting](../resources/crosstenantaccesspolicyb2bsetting.md) | Defines your partner-specific configuration for users from other organizations accessing your resources via Azure AD B2B direct connect. |
 | b2bDirectConnectOutbound | [crossTenantAccessPolicyB2BSetting](../resources/crosstenantaccesspolicyb2bsetting.md) | Defines your partner-specific configuration for users in your organization going outbound to access resources in another organization via Azure AD B2B direct connect. |
 | inboundTrust | [crossTenantAccessPolicyInboundTrust](../resources/crosstenantaccesspolicyinboundtrust.md) | Determines the partner-specific configuration for trusting other Conditional Access claims from external Azure AD organizations. |
+| automaticUserConsentSettings | [inboundOutboundPolicyConfiguration](../resources/inboundoutboundpolicyconfiguration.md) | Determines the partner-specific configuration for automatic user consent settings. |
```
Comment (FaithOmbongi) [inline api-reference/beta/api/crosstenantaccesspolicy-post-partners.md:59]: Move to top of table

File: changelog/Microsoft.DirectoryServices.json [hunk 6]
```diff
@@ -3,6 +3,40 @@
     {
       "ChangeList": [
         {
+          "Id": "b93829d7-509b-43bf-9ff7-b9a17032bae6",
+          "ApiChange": "Resource",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:7]:
```suggestion
          "ApiChange": "Property",
```

File: changelog/Microsoft.DirectoryServices.json [hunk 7]
```diff
@@ -3,6 +3,40 @@
...
+          "ChangedApiName": "automaticUserConsentSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the **automaticUserConsentSettings** property to the [crossTenantAccessPolicyConfigurationDefault](https://docs.microsoft.com/graph/api/resources/crosstenantaccesspolicyconfigurationdefault?view=graph-rest-beta) resource.",
+          "Target": "crossTenantAccessPolicyConfigurationDefault"
+        },
+        {
+          "Id": "b93829d7-509b-43bf-9ff7-b9a17032bae6",
+          "ApiChange": "Resource",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:15]:
```suggestion
          "ApiChange": "Property",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @jkdouglas for your work on this PR. I've left suggestions for minor fixes - maintaining the alphabetical order of the properties tables, shortening the titles for the examples, and changelog fixes.

File: changelog/Microsoft.DirectoryServices.json [hunk 8]
```diff
@@ -3,6 +3,40 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the **automaticUserConsentSettings** property to the [crossTenantAccessPolicyConfigurationDefault](https://docs.microsoft.com/graph/api/resources/crosstenantaccesspolicyconfigurationpartner?view=graph-rest-beta) resource.",
+          "Target": "crossTenantAccessPolicyConfigurationPartner"
+        },
+        {
+          "Id": "b93829d7-509b-43bf-9ff7-b9a17032bae6",
+          "ApiChange": "Resource",
+          "ChangedApiName": "inboundOutboundPolicyConfiguration",
+          "ChangeType": "Addition",
+          "Description": "Added the [inboundOutboundPolicyConfiguration](https://docs.microsoft.com/graph/resources/inboundoutboundpolicyconfiguration?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:26]: This should fix your changelog error
```suggestion
          "Description": "Added the [inboundOutboundPolicyConfiguration](https://docs.microsoft.com/graph/api/resources/inboundoutboundpolicyconfiguration?view=graph-rest-beta) resource.",
```

File: api-reference/beta/resources/crosstenantaccesspolicyconfigurationdefault.md (hunk 3 above)
Comment (jkdouglas) [reply api-reference/beta/resources/crosstenantaccesspolicyconfigurationdefault.md:35]: Fixed.

File: api-reference/beta/api/crosstenantaccesspolicy-post-partners.md (hunk 5 above)
Comment (jkdouglas) [reply api-reference/beta/api/crosstenantaccesspolicy-post-partners.md]: Fixed.

File: api-reference/beta/api/crosstenantaccesspolicyconfigurationpartner-update.md (hunk 4 above)
Comment (jkdouglas) [reply api-reference/beta/api/crosstenantaccesspolicyconfigurationpartner-update.md]: Fixed.

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**